run-qa-checks

```diff
@@ -2,9 +2,12 @@
 
 set -e
 
-# JSLint check
-jslint openwisp_controller/pki/static/pki/js/*.js
-jslint openwisp_controller/config/static/config/js/vpn.js
+# JSHint check
+# jslint openwisp_controller/pki/static/pki/js/*.js
+jshint openwisp_controller/pki/static/admin/pki/js/*.js
+
+# jslint openwisp_controller/config/static/config/js/*.js
```

Comment:
yes you can remove these, no need to comment them, we can see the change by looking at the red line